tfsdk/config_test.go

```diff
@@ -0,0 +1,222 @@
+package tfsdk
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	testtypes "github.com/hashicorp/terraform-plugin-framework/internal/testing/types"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+func TestConfigGet(t *testing.T) {
```

Comment:
Is there a reason not to make this a table-driven test?

Reply:
I was following the style of the other State tests, personally I'd prefer TDTing as well.

Reply:
I'm like 3/4 of the way through converting these... will pick up effort our pairing.

Reply:
Latest commit switched all `Config`, `Plan`, and `State` testing to table driven testing.